### PR TITLE
Caption Character Count

### PR DESCRIPTION
**Pull request by 902seanryan**

### CODE REVIEW

File: src/components/FileExplorer.vue [hunk 1]
```diff
@@ -32,7 +32,8 @@
           <h2 class="font-semi-bold json__dialog-title">Warning</h2>
         </v-card-title>
         <v-card-text>
-          <span class>Importing new files will overwrite any work you have completed so far.</span>
+          <p>Importing new files will remove the currently imported files.</p>
+          <p>You previous work will remain and you will still be able to edit it directly via JSON</p>
```
Comment (chipbell4) [inline src/components/FileExplorer.vue]: "Your" previous work...

File: src/components/TextEditor.vue [hunk 2]
```diff
@@ -6,36 +6,50 @@
...
+              :selected="color.default"
+            />
+          </select>
+          <button class="ql-bold">Bold</button>
+          <button @click="escapeString" class="editor__escape-button">
+            <span v-pre>&#123;&#123; &#125;&#125;</span>
+          </button>
+          </span>
+          <span class="toolbar__group --col">
+            <span v-show="characterCount > 40" class="editor__character-count font-14"><v-icon>warning</v-icon> It is reccomended that caption lines are below 40 characters</span>
```
Comment (chipbell4) [inline src/components/TextEditor.vue]: "It is recommended that caption lines are 40 characters or less"

File: src/components/TextEditor.vue [hunk 3]
```diff
@@ -6,36 +6,50 @@
...
+            <span class="editor__character-count"><span :class="{'yellow--text text--darken-4': characterCount > 40}">{{ characterCount }}</span> / 40</span>
+          </span>
+        </span>
       </div>
     </quill-editor>
     <div class="editor__controls">
+      <div class="editor__controls-error">
+        <!-- > 2 here accounts for the newline that the text editor inserts at the end of the text content-->
+        <span v-show="newLineCount > 2" class="editor__character-count font-14"><v-icon>warning</v-icon> It is reccomended that individual captions have no more than 2 lines</span>
```
Comment (chipbell4) [inline src/components/TextEditor.vue]: "It is recommended that individual captions be no longer than 2 lines"

**Review by chipbell4 — CHANGES_REQUESTED** [review]
Some typos and captions changes

**Comment by 902seanryan** [comment]
With both Errors:
Screen Shot 2020-03-06 at 2 23 22 PM

With one error:
Screen Shot 2020-03-06 at 2 24 28 PM

With no errors:
Screen Shot 2020-03-06 at 2 23 52 PM

**Review by chipbell4 — APPROVED** [review]

**Review by amalley — APPROVED** [review]

**Pull request merged**